elements/bulbs-reading-list/lib/reading-list-article.js

```diff
@@ -106,8 +106,39 @@ export default class ReadingListArticle {
     this.element.dataset.loadStatus = 'loading';
   }
 
+  setupTaboola () {
+    let _taboola = window._taboola || [];
```

Comment:
I think this underscore is an angular thing that might have gotten copied over. you're probably fine just calling this variable `taboola`